### PR TITLE
EG-4067: Fixing lots of links and typos

### PR DESCRIPTION
**Pull request by edward-prosser**
Link checker has been run on the output and there appear to be no broken links!

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/metering-collector.md [hunk 1]
```diff
@@ -854,4 +854,4 @@ All classes listed below belong to the `com.r3.corda.metering.filter` package.
 
 {{% note %}}
 The metering collection functionality Filtering by CorDapp name is case insensitive for MSSQL Server. For more information, see [Database configuration - SQL Server](node/operating/node-database-admin.md#sql-server-3)
-{{% /note %}}
\ No newline at end of file
+{{% /note %}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/metering-collector.md:857]:
```suggestion
{{< /note >}}
```

File: content/en/docs/corda-enterprise/4.6/operations/deployment/host-prereq.md [hunk 2]
```diff
@@ -139,6 +139,6 @@ During deployment the following system (not user) tables will be created in the
 * VAULT_TRANSACTION_NOTES
 * V_PKEY_HASH_EX_ID_MAP
 
-Detailed information on the Corda Vault can be found [here](../operating/node-database.html).
+Detailed information on the Corda Vault can be found [here](../../node/operating/node-database.html).
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/operations/deployment/host-prereq.md:142]:
```suggestion
Detailed information on the Corda Vault can be found [here](../../node/operating/node-database.md).
```

File: content/en/docs/corda-os/4.6/core-tutorials-index.md [hunk 3]
```diff
@@ -17,15 +17,15 @@ title: Core tutorials
...
 | Main steps in basic CorDapp development | Core tutorials |
 |---------|----------|----------|----------|
-| 1. Write the contract and test that it works. | (a) [Writing contracts](tutorial-contract.md) <br/><br/> (b) [Writing contract tests](tutorial-test-dsl.md) |
-| 2. Build your transactions. | (a) [Building transactions](tutorial-building-transactions.md) |
-| 3. Write your flows and test that they work. | (a) [Writing flows](flow-state-machines.md) <br/><br/> (b) [Writing flow tests](flow-testing.md) |
-| 4. Check that it all works together. | (a) [Conducting integration testing](tutorial-integration-testing.md) |
+| 1. Write the contract and test that it works. | (a) [Writing contracts](tutorial-contract.html/) <br/><br/> (b) [Writing contract tests](tutorial-test-dsl.html/) |
+| 2. Build your transactions. | (a) [Building transactions](tutorial-building-transactions.html/) |
+| 3. Write your flows and test that they work. | (a) [Writing flows](flow-state-machines.html/) <br/><br/> (b) [Writing flow tests](flow-testing.html/) |
+| 4. Check that it all works together. | (a) [Conducting integration testing](tutorial-integration-testing.html/) |
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/core-tutorials-index.md:30]: Don't these work as .md links?

File: content/en/docs/corda-os/4.6/supplementary-tutorials-index.md [hunk 4]
```diff
@@ -24,9 +24,9 @@ As these tutorials don’t depend on the code from any of the other tutorials, y
...
-| **Contracts** | [Scheduling events](event-scheduling.md) <br/><br/> [Upgrading contracts](contract-upgrade.md)|
-| **Transactions** | [Working with attachments](tutorial-attachments.md) <br/><br/> [Defining transaction tear-offs](tutorial-tear-offs.md) <br/><br/> [Posting transactions to observer nodes](tutorial-observer-nodes.md) |
-| **Oracles** | [Writing oracle services](oracles.md) |
-| **Notaries** | [Writing custom notary services (experimental)](tutorial-custom-notary.md) |
-| **Nodes** | [Working with the CordaRPCClient API](tutorial-clientrpc-api.md) |
+| **Contracts** | [Scheduling events](event-scheduling.html/) <br/><br/> [Upgrading contracts](contract-upgrade.html/)|
+| **Transactions** | [Working with attachments](tutorial-attachments.html/) <br/><br/> [Defining transaction tear-offs](tutorial-tear-offs.html/) <br/><br/> [Posting transactions to observer nodes](tutorial-observer-nodes.html/) |
+| **Oracles** | [Writing oracle services](oracles.html/) |
+| **Notaries** | [Writing custom notary services (experimental)](tutorial-custom-notary.html/) |
+| **Nodes** | [Working with the CordaRPCClient API](tutorial-clientrpc-api.html/) |
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/supplementary-tutorials-index.md:31]: Same question as above

File: content/en/docs/corda-os/4.6/core-tutorials-index.md (hunk 3 above)
Comment (edward-prosser) [reply content/en/docs/corda-os/4.6/core-tutorials-index.md:30]: No, .md links inside a {{< table >}} tag break.

Comment (ivanterziev-r3) [reply content/en/docs/corda-os/4.6/core-tutorials-index.md:30]: Ah ok, @benpesterr3 can you please add this as a note to the table shortcode description in the KB help?

File: content/en/docs/corda-enterprise/4.6/metering-collector.md (hunk 1 above)
Comment (edward-prosser) [reply content/en/docs/corda-enterprise/4.6/metering-collector.md:857]: Is there a problem with using {{% note %}}?

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.6/metering-collector.md:857]: No, there isn't - but using the % % delimiter requires a full render by the content renderer (Blackfriday) while < > doesn't. I guess this is better for performance.

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**